apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/kafka/listener/KafkaMetaStoreEventListener.java

```diff
@@ -164,15 +161,6 @@ public void onDropDatabase(DropDatabaseEvent dbEvent) {}
   @Override
   public void onLoadPartitionDone(LoadPartitionDoneEvent partSetDoneEvent) {}
 
-  @Override
-  public void onAddIndex(AddIndexEvent indexEvent) {}
-
-  @Override
-  public void onDropIndex(DropIndexEvent indexEvent) {}
-
-  @Override
-  public void onAlterIndex(AlterIndexEvent indexEvent) {}
-
```

Comment:
These no longer exist in hive 3 from what I could tell: https://github.com/apache/hive/tree/rel/release-3.1.2/standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/events

Reply:
OK, we've never done anything with them so that's probably fine, it's just another breaking change in their API.